src/main/java/com/minecolonies/client/gui/AbstractWindowSkeleton.java

```diff
@@ -65,6 +65,7 @@ public void onButtonClicked(Button button)
      */
     public final void doNothing(Button ignored)
     {
+       System.out.print("lalala");
         //do nothing with that event
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Comment has incorrect indentation level 8, expected is 7, indentation should be the same level as line 68. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck)